docs/content-services/dialogs/dialog.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: Dialog component
+Added: v6.9.0
+Status: Active
+Last reviewed: 2024-05-17
+---
+
+# [Dialog component](../../../lib/content-services/src/lib/dialogs/dialog/ "Defined in dialog.component.ts")
+
+Dialog wrapper styled according to a consistent design system.
+
+## Dialog views
+
+### Large size and Medium
+
+Looks the same but have different sizes.
+Max-width for Large dialog is `1075px`;
+Max-width for Medium dialog is `778px`;
+
+![Large and Medium dialog component](../../docassets/images/adf-dialog.png)
+
+### Alert dialogs
+
+Standard:
+
+![Standard alert dialog component](../../docassets/images/adf-dialog-alert-standart.png)
+
+With icon:
+
+![Alert dialog component with icon](../../docassets/images/adf-dialog-alert-with-icon.png)
+
+### Dialog with additional buttons
+
+![Dialog with additional buttons](../../docassets/images/adf-dialog-with-additional-buttons.png)
+
+## Basic Usage 
+
+```html
+
+<ng-template #contentDialogTemplate>
+  Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique nihil, natus corrupti asperiores voluptas, incidunt veritatis.
+</ng-template>
+
+<ng-template #actionsDialogTemplate>
+  <button
+    mat-button
+    adf-auto-focus
+    id="aspect-dialog-actions-cancel"
```

Comment:
for the docs, you probably can omit the `id` and `data-automation-id`

Reply:
yes, you're right, removed it